### ISSUE TITLE
Some splits are completely invisible

### ISSUE THREAD

**Issue by valqelyan** [dm1try/golden_size#3]
```console
NVIM v0.4.4
Build type: Release
LuaJIT 2.0.5
```

Here's how the [golden-ratio](https://github.com/roman/golden-ratio) works for me

https://user-images.githubusercontent.com/14107127/112630797-8b909100-8e4f-11eb-994e-e01b54e01d9a.mp4

Here's how golden_size works

https://user-images.githubusercontent.com/14107127/112630817-93503580-8e4f-11eb-973e-054c79da96e8.mp4

I can't see my windows they seem to have disappeared.

**Comment by dm1try** [comment]
I agree, the original behavior looks more appropriate, I will take a look soon how to achieve this.
